api/v1alpha08/sonataflowplatform_services_types.go

```diff
@@ -19,6 +19,9 @@ type ServicesPlatformSpec struct {
 	// Deploys the Data Index service for use by "prod" profile workflows.
 	// +optional
 	DataIndex *ServiceSpec `json:"dataIndex,omitempty"`
+	// Deploys the Job service for use by "prod" profile workflows.
+	// +optional
+	JobService *ServiceSpec `json:"job,omitempty"`
```

Comment:
didn't we talk about naming this field `jobService` ?
```go
	JobService *ServiceSpec `json:"jobService,omitempty"`
```

Reply:
good catch! I'll rename it as agreed to `jobService`